src/app/domain/sharing/services/SharingNetwork.ts

```diff
@@ -0,0 +1,48 @@
+import CozyClient, { generateWebLink, Q } from 'cozy-client'
+
+import { sharingLogger } from '..'
+import { SharingCozyApp } from '../models/SharingCozyApp'
+
+export const fetchSharingCozyApps = {
+  definition: Q('io.cozy.apps').where({
+    'accept_documents_from_flagship.route_to_upload': { $exists: true },
+    accept_from_flagship: true
+  }),
+  options: {
+    as: 'io.cozy.apps/fetchSharingCozyApps'
```

Comment:
We should not call the query by the functional meaning but by what it does: https://github.com/cozy/cozy-guidelines/#naming-of-queries